integration/coder-app-open-in/main.tf

```diff
@@ -17,18 +17,11 @@ resource "coder_agent" "dev" {
   dir  = "/workspace"
 }
 
-resource "coder_app" "window" {
+resource "coder_app" "tab" {
```

Comment:
The resource name is `tab`, so we the property `coder_app.tab.open_in` is available, but you're referring to `coder_app.slim-window.open_in` hence the issue?

Reply:
Yes indeed , nice catch - just fixed it. 🤔